### PR TITLE
test(functional): add missing scenarios to the delete account and display name test

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
I made the following changes:
- added scenario for "delete account with **incorrect** password" in the delete_account.js, selectors.js file
- made small changes to the "delete account with **correct password** and **click cancel** testcases" in delete_account.js file
- added scenario for 'changing display name' in the settings.js file

@vbudhram - Please review.

### CODE REVIEW

File: packages/fxa-content-server/tests/functional/delete_account.js [hunk 1]
```diff
@@ -51,10 +54,41 @@ registerSuite('delete_account', {
               selectors.SETTINGS_DELETE_ACCOUNT.DETAILS
             )
           )
-          .then(fillOutDeleteAccount(PASSWORD))
+          .findAllByCssSelector(selectors.SETTINGS_DELETE_ACCOUNT.CHECKBOXES)
+          .then(checkboxes => checkboxes.map(checkbox => checkbox.click()))
+          .end()
+          .then(
+            type(selectors.SETTINGS_DELETE_ACCOUNT.INPUT_PASSWORD, 'PASSWORD')
```
Comment (vbudhram) [inline packages/fxa-content-server/tests/functional/delete_account.js:61]: I would probably call the `'PASSWORD'` something like `invalid password` or something similar.

File: packages/fxa-content-server/tests/functional/delete_account.js [hunk 2]
```diff
@@ -51,10 +54,41 @@ registerSuite('delete_account', {
...
+          .then(click(selectors.SETTINGS_DELETE_ACCOUNT.SUBMIT))
+          .then(
+            visibleByQSA(
+              selectors.SETTINGS_DELETE_ACCOUNT.TOOLTIP_INCORRECT_PASSWORD
+            )
+          )
+      );
+    },
+
+    'sign in, delete account with correct password': function() {
```
Comment (vbudhram) [inline packages/fxa-content-server/tests/functional/delete_account.js:72]: I think you can combine the two `sign in, delete account`. In this test case, before you use the correct password, you could use the invalid password.

**Comment by rbillings** [comment]
Suggest the title should include "delete account" so it's easy to know which test changed.

File: packages/fxa-content-server/tests/functional/delete_account.js (hunk 1 above)
Comment (ashrivastava-qa) [reply packages/fxa-content-server/tests/functional/delete_account.js]: I made the suggested change.

File: packages/fxa-content-server/tests/functional/delete_account.js (hunk 2 above)
Comment (ashrivastava-qa) [reply packages/fxa-content-server/tests/functional/delete_account.js]: I made the suggested change.

**Pull request merged**